### ISSUE TITLE
commit "93b08f1" not working

### ISSUE THREAD

**Issue by pedoc** [cefsharp/CefSharp#2193]
I use visual studio 2015(build platfrom win10,win8.1 sdk installed) compiled code for the master branch(commit " #93b08f1"),When I try to run the sample project("CefSharp.WinForms.Example")，The program crashed.Below is the event log, which may help determine the problem,I'm not sure if other people have this problem too

```
application name: CefSharp.WinForms.Example.exe，版本: 60.0.0.0，时间戳: 0x5a07daac
module name: libcef.dll，版本: 3.3071.1649.0，时间戳: 0x596ef916
exception code: 0x4000001f
offset: 0x00665d8c
process ID: 0x5b68
start at: 0x01d35b7650f32e8c
application path: J:\Study\CefSharp\CefSharp.WinForms.Example\bin\x86\Debug\CefSharp.WinForms.Example.exe
module path: J:\Study\CefSharp\CefSharp.WinForms.Example\bin\x86\Debug\libcef.dll
report ID: 1cd58e38-ddb2-45fa-8136-8fa3ff42b06b
```

**Comment by pedoc** [comment]
updated:
i upgraded cef.redist and cef.sdk from v3.3071.1649 to v3.3112.1657,work fine

**Comment by pedoc** [comment]
build myself,published v60.0.0.0 in
https://www.myget.org/F/pedoc/api/v3/index.json
